browser/main/lib/dataApi/copyImage.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs')
+const path = require('path')
+const _ = require('lodash')
+const sander = require('sander')
+
+/**
+ * @description To copy an image and return the path.
+ * @param {String} filePath
+ * @param {String} storageKey
+ * @return {String} an image path
+ */
+function copyImage (filePath, storageKey) {
+  const targetStorage = (() => {
+    try {
+      const cachedStorageList = JSON.parse(localStorage.getItem('storages'))
+      if (!_.isArray(cachedStorageList)) throw new Error('Target storage doesn\'t exist.')
+
+      const storage = _.find(cachedStorageList, {key: storageKey})
+      if (storage == null) throw new Error('Target storage doesn\'t exist.')
```

Comment:
Can't you use `===`?
And maybe, `_.find` returns `undefined`.
https://lodash.com/docs/4.17.4#find
So you need to write `=== undefined`.